core/src/main/scala/org/apache/spark/sql/delta/commands/UpdateCommand.scala

```diff
@@ -51,6 +51,10 @@ case class UpdateCommand(
     condition: Option[Expression])
   extends LeafRunnableCommand with DeltaCommand {
 
+  override val output: Seq[Attribute] = {
+    Seq(AttributeReference("num_updated_rows", LongType)())
```

Comment:
I should've clarified this in the issue, my bad.

Can you make this `num_affected_rows` please? To be consistent with the DELETE command output.

https://github.com/delta-io/delta/blob/master/core/src/main/scala/org/apache/spark/sql/delta/commands/DeleteCommand.scala#L86